tests/runtime/test_service_factory.py

```diff
@@ -146,7 +148,146 @@ def test_get_and_filter_modules_respects_included_task_types_and_excluded_module
             }
         }
     ) as cfg:
-
         clean_modules = ServicePackageFactory._get_and_filter_modules(cfg, "sample_lib")
         assert "SampleModule" in str(clean_modules)
         assert "ListModule" not in str(clean_modules)
+
+
+def get_packages_with_override(
```

Comment:
NIT: We usually try to co-locate helper files like this into a `## Helpers ##` section header at the top